camptocamp/ngeo
# Add webpack generation

Removes @module 
Add goog.asserts

Done:
* development build
* production build
* CSS build
* tests
* tests in debug mode
* source map in development build
* source map in production build

TODO:
* only the search example is build it should be generalized
* development server
* coverage
* add all missing externs requirement
* add import css
* manage partials
* the dist
* single build for the tests?

Required that all the files are at the right place to uncomment the test run...
